example-get-started/generate.sh

```diff
@@ -12,11 +12,64 @@ REPO_NAME="example-get-started"
 REPO_PATH="$HERE/build/$REPO_NAME"
 PROD=${1:-false}
 
+# Some additional flags to tune the exact repo structure that we generate.
+# It useful to generate nested (monorepo), private storage, a mix of those cases
+# to be used in Studio tests or QA.
+TESTING_REPO='false' # Default false.
+INIT_GIT='true' # Default true.
+INIT_DVC='true' # Default true.
+NON_DVC='false' # Default false.
+BRANCHES='true' # Default true.
+REMOTE="public-s3" # Default "private-s3". Other options: "public-s3", "private-http", "private-ssh", etc.
+DVC_TRACKED_METRICS='false' # Default false.
+SUBDIR='' # No leading or trailing slashes. Default "".
+
+if [ -z $SUBDIR ]; then
+  COMMIT_PREFIX=""
+  GIT_TAG_SUFFIX=""
+  GTO_PREFIX=""
+else
+  [ -d "$REPO_PATH" ] && cp -r "$REPO_PATH" "$REPO_PATH-backup-$(date +%s)"
+  MODIFIER=$(echo ${SUBDIR} | tr / -)
+  COMMIT_PREFIX="[$MODIFIER] "
+  GIT_TAG_SUFFIX="-$MODIFIER"
+  # In GTO we use : as a separator to get the full model name
+  GTO_PREFIX="${SUBDIR}:"
+fi
+
+REPO_PATH="${HERE}/build/${REPO_NAME}/${SUBDIR}"
 if [ -d "$REPO_PATH" ]; then
   echo "Repo $REPO_PATH already exists, please remove it first."
   exit 1
 fi
 
+
+init_remote_storage() {
+  if [ $REMOTE == 'public-s3' ]; then
+    # Remote active on this env only, for writing to HTTP redirect below.
+    dvc remote add -d --local storage s3://dvc-public/remote/get-started
+    # Actual remote for generated project (read-only). Redirect of S3 bucket above.
+    dvc remote add -d storage https://remote.dvc.org/get-started
+  fi
+
+  if [ $REMOTE == 'private-s3' ]; then
+    # Remote active on this env only, for writing to HTTP redirect below.
+    dvc remote add -d storage s3://dvc-private/remote/get-started
+  fi
+
+  if [ $REMOTE == 'private-http' ]; then
+    dvc remote add -d --local storage ssh://dvc@35.194.53.251/home/dvc/storage
+    dvc remote modify --local storage keyfile /Users/ivan/.ssh/dvc_gcp_remotes_rsa
```

Comment:
This makes it only generable by you.
I guess it is ok if it is not mean to be generated frequently

Reply:
> This makes it only generable by you.

Yep, I'll add the details to the README for this repo. Ppl can manually change the path for now, not critical.

Reply:
added the README and link to the Slack thread with the credentials